### PR TITLE
Polish translation reviewed (in major part done by @w1stler)

### PR DESCRIPTION
**Pull request by magul**

### CODE REVIEW

File: pl/python_introduction/README.md [hunk 1]
```diff
@@ -773,7 +773,7 @@ else:
     print('Hej anonimie!')
 ```
 
-i uruchommy go:
+i uruchomimy go:
```
Comment (w1stler) [inline pl/python_introduction/README.md]: Might be 'uruchommy go', as in other parts

Comment (magul) [reply pl/python_introduction/README.md]: That's exactly how it looks on Crowdin, so if You want to fix it, do it there. :)

Comment (w1stler) [reply pl/python_introduction/README.md]: Done.

**Comment by magul** [comment]
I just force-pushed newest build from Crowdin so feel welcomed to review it once again @w1stler

**Pull request merged**